packages/paste-website/src/pages/patterns/empty-states/index.mdx

```diff
@@ -0,0 +1,444 @@
+---
+title: Empty states
+description: Give customers guidance on what steps to take if content is either unavailable or can't be displayed.
+slug: /patterns/empty-states/
+---
+
+import {graphql} from 'gatsby';
+import {Anchor} from '@twilio-paste/anchor';
+import {Box} from '@twilio-paste/box';
+import {Breadcrumb, BreadcrumbItem} from '../../../components/breadcrumb';
+import {Callout, CalloutTitle, CalloutText} from '../../../components/callout';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Card} from '@twilio-paste/card';
+import {Grid, Column} from '@twilio-paste/grid';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Table, Tr, Th, Td, THead, TBody} from '@twilio-paste/table';
+import {Text} from '@twilio-paste/text';
+import {useUID} from 'react-uid';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/patterns/empty-states/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<content>
+
+<Breadcrumb>
+  <BreadcrumbItem to="/">Home</BreadcrumbItem>
+  <BreadcrumbItem to="/patterns">Patterns</BreadcrumbItem>
+</Breadcrumb>
+<Heading as="h1" variant="heading10">
+  {props.pageContext.frontmatter.title}
+</Heading>
+<Paragraph>{props.pageContext.frontmatter.description}</Paragraph>
+
+<Box as="dl" display="flex">
+  <Text as="dt" color="colorTextWeak" lineHeight="lineHeight30">
+    <Box width="size10">Status</Box>
+  </Text>
+  <Text as="dd">beta</Text>
+</Box>
+
+</content>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Ingredients
+
+<Grid gutter="space30" marginBottom="space80">
+  <Column>
+    <Card>
+      <Heading as="h3" variant="heading40" marginBottom="space0">
+        <Anchor href="../components/card">Card</Anchor>
+      </Heading>
```

Comment:
Can you add a paragraph to this section that says

> Go to the [Sizing & responsiveness section](link to sizing & responsiveness) for more details.

Reply:
@serifluous Does this work? https://share.getcloudapp.com/4gu2Ze8B

I tried adding the copy to the Card and Illustration boxes, but that really throws off the box layout.